spec/models/company_spec.rb

```diff
@@ -34,4 +34,47 @@
       expect(company).to respond_to(:jobs)
     end
   end
+
+  describe "company methods" do
```

Comment:
i like that you have tests for these methods 👍 can you DRY up any of this data prep?